Dockerfile

```diff
@@ -42,20 +42,31 @@ ARG TARGETARCH
 
 # Install runtime OS package dependencies
 RUN --mount=type=cache,target=/var/cache/apt \
-    apt-get update && \
+    apt update && \
```

Comment:
Actually, apt does not have a stable API, which is why we used `apt-get` here.

Reply:
Gotcha, thought that `apt` was the standard these days. Will revert :+1: